hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -1106,6 +1117,10 @@ public PartialUpdateMode getPartialUpdateMode() {
     }
   }
 
+  public String getMergeProperties() {
```

Comment:
is this another encoding of k-v property pairs into a string.. 

if so - see my comments here. 

https://github.com/apache/hudi/pull/13498#discussion_r2224118191

can we also fix this and move all custom merge properties to `hoodie.merge.custom.*` prefix.. and handle that in upgrade/downgrade as well. This is a separate, but blocker PR.

Reply:
Addressing it as part of https://github.com/apache/hudi/pull/13616